core/kazoo_globals/src/kz_nodes.erl

```diff
@@ -78,11 +83,16 @@
 -define(SIMPLE_ROW_STR, ?HEADER_COL ": ~s~n").
 -define(SIMPLE_ROW_NUM, ?HEADER_COL ": ~B~n").
 
+-define(NODE_GEN_SECRET, kz_binary:rand_hex(16)).
```

Comment:
why do you want to do that ? this is a way to provide immediate correlation not time basis.